### ISSUE TITLE
Hyperparameter tuning & evaluation (Primary Model)

### ISSUE THREAD

**Issue by mariafnafees** [mariafnafees/APS360_Project#7]
Selection and reporting of accuracy/loss results from training/validation/testing of Primary model.
Varying selection of:
- Batch Size
- Iteration number
- Learning Rate
- Optimizer
- Momentum
